lib/IRGen/GenStruct.cpp

```diff
@@ -290,7 +291,46 @@ namespace {
       }
     }
   };
-  
+
+  class ClangRecordAddressOnlyTypeInfo final
```

Comment:
Could you move it after `ClangRecordTypeInfo` to keep the order in the enum and here identical?